src/pages/using-spark/guides/alt-text.mdx

```diff
@@ -0,0 +1,352 @@
+---
+title: Alternative Text
+---
+
+import { SprkDivider, SprkIcon } from '@sparkdesignsystem/spark-react';
+
+<h1 class="sprk-b-PageTitle sprk-b-TypeDisplayOne sprk-u-mbl sprk-u-Measure">
+  Alternative Text
+</h1>
+
+UX Writers, designers and developers can use this **Alternative Text Guide** as an instruction to writing style, usage, text for copy, documentation, reference information, and copy for all images inside Rocket Products digital experiences.
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvl" />
+
+<h3 class="sprk-b-TypeDisplayThree sprk-u-mbm sprk-u-Measure">Main Takeaways</h3>
+
+- Every image's alternative text should describe the image without altering the meaning of the screen or referencing the word “image.”
+- When writing copy for alternative text, imagine what you would say when reading the screen to someone over the phone without mentioning that there is an image on the screen.
+- Adding alternative text ensures users with cognitive, learning, visual impairments and other disabilities can understand non-text content.
+- Use a null attribute (or empty string) for non-semantic or decorative images, i.e., those that are for ambiance or other non-semantic purposes.
+
+#### Accessibility Benefits
+
+- Screen readers read alt-text in place of images. This allows the image to be accessible to users with visual impairment.
+- When an image fails to load or if content is blocked, the alt-text will appear in it's place.
+- It gives images a text to display so search engines can read it without altering the meaning of the screen.
```

Comment:
is this about search engines?  or screen readers.

Reply:
screen readers :)